src/bitcoin-cli.cpp

```diff
@@ -197,7 +198,33 @@ static void http_error_cb(enum evhttp_request_error err, void *ctx)
 }
 #endif
 
-UniValue CallRPC(const std::string& strMethod, const UniValue& params)
+tl::expected<UniValue, UniValue> CallRPC(const std::string& strMethod, const UniValue& params);
+
+UniValue ConvertValues(const std::string &strMethod, const std::vector<std::string> &strParams)
+{
+    return RPCConvertValues(strMethod, strParams)
+        .map_error([&](auto failure) {
+            throw std::runtime_error(
+                    FormatConversionFailure(strMethod, failure)
+                    + std::visit(match {
```

Comment:
This can be `examine(failure, match { ...`.

Reply:
Ah, thanks – I’ve been scanning my PRs for places that I could update with `examine`, but apparently missed this one.